manifests/administration.pp

```diff
@@ -30,14 +30,43 @@
       value  => '1',
     }
   }
+  file { '/etc/systemd/system/disable_thp.service':
```

Comment:
The $disable_thp condition got lost and hugepages are now disabled by default. Please add it back

Reply:
Done